Eastern-Research-Group/mywaterway
# Feature/854 configurable text

## Related Issues:
* [HMW-854](https://jira.epa.gov/browse/HMW-854)

## Main Changes:
* Moved the text on the Extreme Weather tab to S3-sourced config.
* Moved the "upper content" on Community tabs to S3 config.
* Fixed a bug that caused the EPA template mobile draw to be illegible.
* Fixed incorrect designation for the RPS service on the Data page.

## Steps To Test:
1. Go to http://localhost:3000/community/dc/extreme-weather. Expand all "Show more" sections and verify the content is correct.
2. For each of the other tabs, expand the "Show more" section in the upper content and verify the content is correct.
